site/app/libraries/database/DatabaseQueries.php

```diff
@@ -4818,7 +4820,8 @@ public function createGradeable(\app\models\gradeable\Gradeable $gradeable) {
                   eg_precision,
                   eg_peer_grading,
                   eg_peer_grade_set,
-                  eg_regrade_request_date,
+                  eg_grade_inquiry_start_date,
+                  eg_grade_inquiry_due_date,
```

Comment:
You need to add anonther .? to the VALUES statement below this INSERT, since you are adding an additional column to insert

Reply:
Yeah actually It was breaking the `create-new-gradeable` flow... thanks!